### ISSUE TITLE
Tabs should be the same color

### ISSUE THREAD

**Issue by ShendMikullovci** [serge-web/serge#1370]
The tabs on the top left window of the mapping page (namely, Control and Visibility) should be the same color since color contrasts distract the user.

![Top Left Window Mapping](https://user-images.githubusercontent.com/85491794/156803096-6e3fcc64-2983-4bf0-8790-a1f84eff371d.png)

I suggest we stick to the "Visibility" tab color. Sticks out more from the background and from the "Submit Routes" button in the window.

Alternatively, we could try to have the selected tab take on the bolder dark blue color alongside the downward arrow to emphasise selection.
